src/general_tally.cpp

```diff
@@ -206,6 +206,98 @@ void GeneralTally::score_source(const BankedParticle& p) {
   tally_gen_score_.element(indices.begin(), indices.end()) += source_score;
 }
 
+// method to evaluate the tally
+double GeneralTally::evaluate(const Position r, const double E) const {
```

Comment:
I am realizing that once we add an `energy_out_` filter in this class, we will also need Eout to be able to evaluate the tally... We need to think about how we want to handle this...